docs/reference/toc.yml

```
@@ -0,0 +1,12 @@
+toc:
+      - file: index.md
+        children:
+          - file: setup/index.md
+            children:
+              - file: setup/k8s.md
+          - file: configuration.md
+          - file: supported-technologies.md
+          - file: metrics.md
+          - file: migration.md
+          - title: Troubleshooting
+            crosslink: docs-content://troubleshoot/ingest/opentelemetry/edot-sdks/nodejs/index.md
```

I think you need to flatten the nav structure (similar to https://github.com/elastic/elastic-agent/pull/9953#pullrequestreview-3226598623):

```suggestion
toc:
      - file: index.md
      - file: setup/index.md
        children:
          - file: setup/k8s.md
      - file: configuration.md
      - file: supported-technologies.md
      - file: metrics.md
      - file: migration.md
      - title: Troubleshooting
        crosslink: docs-content://troubleshoot/ingest/opentelemetry/edot-sdks/nodejs/index.md
```